epregressions/runtests.py

```diff
@@ -73,7 +73,10 @@ def __init__(self, run_config, these_entries, mute=False):
         self.diff_completed_callback = None
         self.all_done_callback = None
         self.cancel_callback = None
+
+        # set a few things
         self.id_like_to_stop_now = False
+        self.completed_structure = None
```

Comment:
Completed structure instance is now a member variable on the suite runner class so that the process pool can access it when processes complete.